catchpoint/config.yml

```
@@ -0,0 +1,49 @@
+# Name of the quickstart
+name: Catchpoint- uickstart
```

I think this might just be a mistype!
```suggestion
name: catchpoint-quickstart
```